pages/app-layout/with-drawers.page.tsx

```diff
@@ -31,10 +37,14 @@ const getAriaLabels = (title: string, badge: boolean) => {
 export default function WithDrawers() {
   const { urlParams, setUrlParams } = useContext(AppContext as DemoContext);
   const [activeDrawerId, setActiveDrawerId] = useState<string | null>(null);
+  const [activePublicDrawerId, setActivePublicDrawerId] = useState<string | null>('pro-help');
   const hasTools = urlParams.hasTools ?? false;
-  const hasDrawers = urlParams.hasDrawers ?? true;
+  const hasBetaDrawers = urlParams.hasBetaDrawers ?? false;
+  const hasPublicDrawers = urlParams.hasPublicDrawers ?? true;
 
-  const drawers = !hasDrawers
+  const drawerLabels = { drawers: 'Drawers', drawersOverflow: 'Drawers overflow' };
+
+  const betaDrawers = !hasBetaDrawers
```

Comment:
I think instead of the parameter we should move beta usage on separate page `with-drawers-beta` and prepare to remove this after the beta expires

Reply:
I wanted to double check that publicDrawers would override beta ones, but I agree.